README.md

```diff
@@ -18,6 +18,7 @@ and loaders, and based on the [Meltano PostgreSQL target].
 In order to learn more about Singer, Meltano, and friends, navigate to the
 [Singer Intro](./docs/singer-intro.md).
 
+Operating the package successfully needs CrateDB 6.2 or higher.
```

Comment:
@surister: Let me loop your comment into a thread.

> Should we perhaps maintain this and check for CrateDB version? I assume that this will break people trying to use it with CrateDB < 6.2.

Reply:
This patch includes a check for the CrateDB version, as suggested.

- GH-65